ourjapanlife/findadoc-server
# 100 sql to graphql: Degrees

Part of #100


# What changed
- Refactor Degrees to match SQL
- Add queries
- Add a test

# Testing instructions

Hey! We have a test runner in here which means #78 is already in progress 🙌🏻 

Let's start writing tests to prove our tickets are working. Then you can run with `yarn test` to verify the real graphQL query works
